feature-toggle-api/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
all of this should be in the charts sub directory
see here: https://github.com/hmcts/bulk-scan-orchestrator/pull/180/files#diff-f1b180f33c542b093bd77bfd83fc7309

(PR not merged unfortunately but we'll get back to it when we can)